src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -779,10 +786,25 @@ public void getBlUsersSelf(List<String> include, boolean expandPerms,
   }
 
   @Override
-  public void postBlUsersLogin(boolean expandPerms, List<String> include, String userAgent, String xForwardedFor,//NOSONAR
+  public void postBlUsersLoginWithExpiry(boolean expandPerms, List<String> include, String userAgent, String xForwardedFor,
+                               LoginCredentials entity, Map<String, String> okapiHeaders,
+                               Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler,
+      Context vertxContext) {
+    doPostBlUsersLogin(expandPerms, include, userAgent, xForwardedFor, entity, okapiHeaders, asyncResultHandler, LOGIN_ENDPOINT);
+  }
+
+  @Override
+  public void postBlUsersLogin(boolean expandPerms, List<String> include, String userAgent, String xForwardedFor,
                                LoginCredentials entity, Map<String, String> okapiHeaders,
                                Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler,
       Context vertxContext) {
+    doPostBlUsersLogin(expandPerms, include, userAgent, xForwardedFor, entity, okapiHeaders, asyncResultHandler, LOGIN_ENDPOINT_LEGACY);
+  }
+
+  private void doPostBlUsersLogin(boolean expandPerms, List<String> include, String userAgent, String xForwardedFor, //NOSONAR
```

Comment:
Replacing NOSONAR with a specific `@SuppressWarnings` doesn't suppress other warnings that we may introduce in future:
https://github.com/folio-org/raml-module-builder/blob/v34.1.0/domain-models-runtime/src/main/java/org/folio/rest/tools/utils/NetworkUtils.java#L19

Reply:
I agree that NOSONAR is a bit broad, but it's  used all over in this class and was suppressing the warning for the this method prior to my small changes.